README.md

```diff
@@ -7,13 +7,14 @@ and is optimized for phones and tablets of all shapes and sizes.
 
 ## Dependencies
 
-1. Jetpack Compose
-2. Coroutines - For Concurrency and Asynchronous tasks
-3. Ktor - For network requests
-4. Hilt - For Dependency Injection
-5. Crashlytics
-6. Coil - For Image Loading and Caching
-7. Lint Checks - [Ktlint](https://ktlint.github.io/)
+1. JAVA 11 - [Install](https://www.oracle.com/java/technologies/javase/jdk11-archive-downloads.html)
```

Comment:
I wouldn't say it is a dependency per sè. I would add it under `Running the Project` and then add steps, for different environments(Mac, Windows, Linux) to check if Gradle is using java 11

Reply:
ok. Let me adjust the changes